content/lessons/chapter-4/public-key-3.tsx

```diff
@@ -46,12 +44,12 @@ const javascript = {
 const python = {
   program: `print("KILL")`,
   defaultFunction: {
-    name: 'find_hash',
-    args: ['nonce'],
+    name: 'rawPublicKey',
```

Comment:
This is the default name of the function that will be displayed on the terminal. In the Figma file, this was written as privatekey_to_publickey .